MiniApp/Classes/core/MiniApp.swift

```diff
@@ -128,6 +128,25 @@ public class MiniApp: NSObject {
     }
 }
 
+// MARK: - UI
+public extension MiniApp {
+    func launch(
```

Comment:
Some documentation may be for the public API ?

Reply:
@ts-leojoseph-r thanks! sure I'll provide more documentation tomorrow. I've just pushed a commit with the changes related to moving the launch stuff to the ui component. 
For the UI part I'd suggest moving on from xibs or storyboards because imho coded ui is easier to read and much easier to maintain / compare etc #discuss :D. (@Climbatize what do you think)
_If SwiftUI will improve in the future I think also that's the way to go but not until apple fixes some stuff_

Reply:
@leorjoseph @Climbatize is it ok?